src/main/java/com/squareup/kotlinpoet/ParameterSpec.kt

```diff
@@ -31,10 +31,13 @@ class ParameterSpec private constructor(builder: ParameterSpec.Builder) {
   val defaultValue = builder.defaultValue
 
   @Throws(IOException::class)
-  internal fun emit(codeWriter: CodeWriter) {
+  internal fun emit(codeWriter: CodeWriter, emitType: Boolean = true) {
```

Comment:
I think I would prefer this being called `includeType` as the "emit" prefix is semi-redundant with the name of the method.